### DISCUSSION TITLE
GitHub LTI v1.1

### DISCUSSION THREAD

**Discussion by ed-cu** [community/community#48428] [GitHub Education]
Hello,

I am a Canvas LMS admin. I had previously integrated individual courses with the instructor's GitHub classroom. I no longer find anything about LTI v1.1 on the GitHub website (admin or integration guide). I had used a config URL and shared secret/consumer key from the instructor.

I'd like to know if there is a way for us to continue to provide support (and for that I need configuration info that's valid), for the course-level v1.1 integration. I am aware of the v1.3 LTI. This is the only link I find regarding Canvas LMS integration, but it only refers to steps for v1.3. https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/register-a-learning-management-system-with-github-classroom

Any help is appreciated.

Regards,

Ed

**Comment by ed-cu** [comment]
Bump

**Comment by samus-aran** [comment]
Hi @ed-cu 👋 

If you still have the existing configuration URL and a generated secret and key they should continue to work with the LTI 1.1 integration. However, if you require to re-authenticate we are not able to provide these steps as we [no longer support](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/connect-a-learning-management-system-course-to-a-classroom#supported-lmses) LTI 1.1. We would recommend users to upgrade to LTI 1.3 to obtain better security with intergrations.

**Comment by ed-cu** [comment]
Thanks very much.

Regards,

Ed
